tests/main/snap-env/task.yaml

```diff
@@ -35,8 +35,8 @@ execute: |
     "$NAME".env | grep -E '^XDG_' | sort > xdg-vars.txt
     "$NAME".env | grep -E '^EXTRA_' | sort > extra-vars.txt
 
-    echo "Collect PATH and HOME environment variables"
-    "$NAME".env | grep -E '^(SNAP|PATH|HOME)=' | sort > misc-vars.txt
+    echo "Collect PATH, HOME and SHELL environment variables"
+    "$NAME".env | grep -E '^(SNAP|PATH|HOME|SHELL)=' | sort > misc-vars.txt
```

Comment:
hm the test doesn't really check that the SHELL and PATH are replaced with different values. Maybe you can add this check at the end?

Reply:
Yeah, that's fair. I'll add one with `SHELL=/bin/sh` perhaps.

Reply:
Done, have a look